### PR TITLE
Anf 567 anf swagger restapi to2020 02 01 no snapshotpolicy

### PR DESCRIPTION
**Pull request by audunn**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by audunn** [comment]
Can we get this looked at?

File: specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-02-01/examples/Accounts_CreateOrUpdate.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "subscriptionId": "D633CC2E-722B-4AE1-B636-BBD9E4C60ED9",
+    "resourceGroupName": "myRG",
+    "accountName": "account1",
+    "api-version": "2019-11-01",
```
Comment (chiragg4u) [inline specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-02-01/examples/Accounts_CreateOrUpdate.json:6]: API version still old, please check other examples also.

**Review by chiragg4u — COMMENTED** [review]
Changes looks good, please check the API versions in the examples. Some of them are referring to older version.

**Comment by audunn** [comment]
> Changes looks good, please check the API versions in the examples. Some of them are referring to older version.

Thank you @chiragg4u  for this catch, this has been fixed.
Funny how it was not found earlier perhaps such things could be a candidate for the validation done in oav validate-example.

**Comment by audunn** [comment]
@chiragg4u can you have another look please?

**Comment by audunn** [comment]
Can we please get this moving urgently needed changes with customers waiting

**Review by pilor — APPROVED** [review]

**Comment by audunn** [comment]
Can we get this merged?

**Comment by audunn** [comment]
Can this be merged urgently needed by customers

**Comment by audunn** [comment]
@zikalino @chiragg4u   Can we get this merged?  
If there is something preventing it from being moved forward can you please advice?

File: specification/netapp/resource-manager/readme.md [hunk 2]
```diff
@@ -111,7 +120,6 @@ swagger-to-sdk:
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-java
-  - repo: azure-sdk-for-trenton
```
Comment (akning-ms) [inline specification/netapp/resource-manager/readme.md:114]: Can we know why remove this entry?

File: specification/netapp/resource-manager/readme.java.md [hunk 3]
```diff
@@ -24,6 +25,18 @@ batch:
...
+These settings apply only when `--tag=package-netapp-2020-02-01 --java` is specified on the command line.
+Please also specify the `--azure-libraries-for-java-folder=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == 'package-netapp-2020-02-01' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.netapp.v2020_02_01
+  output-folder: $(azure-libraries-for-java-folder)/sdk/netapp/mgmt-v2020_02_01
+regenerate-manager: true
+generate-interface: true
+```
```
Comment (akning-ms) [inline specification/netapp/resource-manager/readme.java.md:40]: + @weidongxu-microsoft  to take a look this change for Java SDK impact

Comment (weidongxu-microsoft) [reply specification/netapp/resource-manager/readme.java.md:40]: LGTM

It is same as the last one `### Tag: package-netapp-2019-11-01 and java`, with change on version. So generally it is good.

File: specification/netapp/resource-manager/readme.md (hunk 2 above)
Comment (audunn) [reply specification/netapp/resource-manager/readme.md:114]: Hi, I missed that entry, I am not sure what it is can I ask what is it for?
Let me know if it is needed and I will add it again.

Comment (akning-ms) [reply specification/netapp/resource-manager/readme.md:114]: Do you need TerraForm? it yes. you need add it back.

Comment (audunn) [reply specification/netapp/resource-manager/readme.md:114]: Thank you, I added it back.

**Pull request merged**